tools/dvrescue.xsd

```diff
@@ -44,36 +50,189 @@
       <xsd:element name="sta" type="staType" minOccurs="0" maxOccurs="unbounded"/>
       <xsd:element name="aud" type="audType" minOccurs="0" maxOccurs="unbounded"/>
     </xsd:sequence>
-    <xsd:attribute name="n" type="xsd:integer"/>
-    <xsd:attribute name="pts" type="xsd:string"/>
-    <xsd:attribute name="tc" type="xsd:string"/>
-    <xsd:attribute name="tc_r" type="xsd:boolean"/>
-    <xsd:attribute name="tc_nc" type="xsd:boolean"/>
-    <xsd:attribute name="rdt" type="xsd:string"/>
-    <xsd:attribute name="rdt_r" type="xsd:boolean"/>
-    <xsd:attribute name="rdt_nc" type="xsd:boolean"/>
-    <xsd:attribute name="rec_start" type="xsd:boolean"/>
-    <xsd:attribute name="arb" type="xsd:string"/>
-    <xsd:attribute name="arb_r" type="xsd:boolean"/>
-    <xsd:attribute name="arb_nc" type="xsd:boolean"/>
+    <xsd:attribute name="n" type="xsd:integer">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          The number of the DIF sequence within the frame starting from zero.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="pts" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Presentation timestamp for frame element.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="tc" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Timecode.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="tc_r" type="xsd:boolean">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Boolean that indicates if timecode is repeating.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="tc_nc" type="xsd:boolean">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Boolean that indicates if timecode is non-consecutive.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="rdt" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Recorded Date Time.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="rdt_r" type="xsd:boolean">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Boolean that indicates if recorded date time is repeating.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="rdt_nc" type="xsd:boolean">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Boolean that indicates if recorded date time is non-consecutive.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="rec_start" type="xsd:boolean">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Recording start.
```

Comment:
This is a flag to say that the frame is the first frame of a recording, as in the camera operator just hit the record button.